components/modal.css

```diff
@@ -0,0 +1,56 @@
+@import "https://mantra-ui.netlify.app/style.css";
+
+.show-modal-btn {
+  width: max-content;
+}
+
+.modal-container {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  background-color: rgba(0, 0, 0, 0.3);
+  display: flex;
+  justify-content: center;
+  padding: 8rem;
+  align-items: start;
+  opacity: 0;
+  pointer-events: none;
+  backdrop-filter: blur(1px);
+}
+
+.modal-container.show {
+  opacity: 1;
+  pointer-events: auto;
+}
+
```

Comment:
Can give higher z-index to the modal container!

Reply:
Thanks for pointing this out. Resolved it.